### PR TITLE
fix: base fix struct

### PR DESCRIPTION
**Pull request by wGoDev**

### CODE REVIEW

**Comment by WahaWaher** [comment]
...

File: package.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "pr-contra",
-  "version": "1.0.0",
```
Comment (wGoDev) [inline package.json:3]: jhjhkl

**Review by WahaWaher — COMMENTED** [review]
comment...

**Review by WahaWaher — APPROVED** [review]
approval... @wGoDev

**Review by WahaWaher — CHANGES_REQUESTED** [review]
please change this...

File: package.json [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-  "main": "index.js",
+  "version": "0.1",
```
Comment (WahaWaher) [inline package.json]: pls, change version

File: package.json [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
   "repository": "https://github.com/WahaWaher/pr-contra",
   "author": "WahaWaher <wahawaher@gmail.com>",
-  "license": "MIT"
+  "license": "MIT",
```
Comment (WahaWaher) [inline package.json]: pls change LiCENSE

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (WahaWaher) [inline .gitignore:1]: pls use standart .gitignore

**Review by WahaWaher — CHANGES_REQUESTED** [review]
3 **changes** pls

File: yarn.lock [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (WahaWaher) [inline yarn.lock:1]: why this?

**Review by WahaWaher — CHANGES_REQUESTED** [review]
dfgdfgdfgdfg

File: package.json (hunk 2 above)
Comment (wGoDev) [reply package.json]: ok

File: package.json (hunk 3 above)
Comment (wGoDev) [reply package.json]: ok

File: .gitignore (hunk 4 above)
Comment (wGoDev) [reply .gitignore:1]: no

File: yarn.lock (hunk 5 above)
Comment (wGoDev) [reply yarn.lock:1]: ...

**Review by WahaWaher — APPROVED** [review]

**Review by WahaWaher — CHANGES_REQUESTED** [review]
...

**Pull request closed without merging**